core/modules/comment/config/views.view.manage_comments.json

```
@@ -0,0 +1,1547 @@
+                            "trim_whitespace": 0,
+                            "alt": "",
+                            "rel": "",
+                            "link_class": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "target": "",
+                            "nl2br": 0,
+                            "max_length": "",
+                            "word_boundary": 1,
+                            "ellipsis": 1,
+                            "more_link": 0,
+                            "more_link_text": "",
+                            "more_link_path": "",
+                            "strip_tags": 0,
+                            "trim": 0,
+                            "preserve_tags": "",
+                            "html": 0
+                        },
+                        "element_type": "",
+                        "element_class": "",
+                        "element_label_type": "",
+                        "element_label_class": "",
+                        "element_label_colon": 1,
+                        "element_wrapper_type": "",
+                        "element_wrapper_class": "",
+                        "element_default_classes": 1,
+                        "empty": "",
+                        "hide_empty": 0,
+                        "empty_zero": 0,
+                        "hide_alter_empty": 1,
+                        "text": "",
+                        "link_to_node": 0
+                    },
+                    "dropbutton": {
+                        "id": "dropbutton",
+                        "table": "views",
+                        "field": "dropbutton",
+                        "relationship": "none",
+                        "group_type": "group",
+                        "ui_name": "",
+                        "label": "Operations",
+                        "exclude": 0,
+                        "alter": {
+                            "alter_text": false,
+                            "text": "",
+                            "make_link": false,
+                            "path": "",
+                            "absolute": false,
+                            "external": false,
+                            "replace_spaces": false,
+                            "path_case": "none",
+                            "trim_whitespace": false,
+                            "alt": "",
+                            "rel": "",
+                            "link_class": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "target": "",
+                            "nl2br": false,
+                            "max_length": "",
+                            "word_boundary": true,
+                            "ellipsis": true,
+                            "more_link": false,
+                            "more_link_text": "",
+                            "more_link_p
```

```suggestion
                        "content": "Number of unapproved comments</em>: @total"
```

Same as previous comment: weird to have the dot at the end of the count, and italics not fitting here.

---

...hmm, something weird going on with Github PR suggestions.